homeassistant/components/auth/__init__.py

```diff
@@ -275,6 +275,12 @@ def __init__(self, retrieve_credentials):
             }, status_code=400)
 
         user = await hass.auth.async_get_or_create_user(credentials)
+
+        if not user.is_active:
+            return self.json({
+                'error': 'invalid_request',
```

Comment:
invaid_auth, 401?

Reply:
These are the possible error codes

https://tools.ietf.org/html/rfc6749#section-4.1.2.1

I guess we can use `access_denied`. We can also `error_description` to add a bit more meaningful info.